assembly/assembly-root-war/src/main/webapp/_app/keycloackLoader.js

```diff
@@ -110,8 +110,8 @@ export class KeycloakLoader {
                     onLoad: 'login-required',
                     checkLoginIframe: false,
                     useNonce: useNonce,
-                    scope: 'email profile',
-                    redirectUri: keycloakSettings['che.keycloak.redirect_url.ide']
+                    scope: 'openid',
```

Comment:
Can you explain the changes in scope. Why it is needed? Is there a change in JWT token content?

Reply:
I don't think, that I changed scope. Keycloak 6.0.1 adapters ignored this option from workspace loader and set up `openid` scope by default. New keycloak adapters respect this option. So I only set up the same value like it actually was before. Also in the get token response with old and new keycloak I see in the json scope field: 'openid email profile'. When I decode jwt token from new keycloak I really see fields:

...
  "scope": "openid email profile",
  "sid": "4d9e49e6-b648-40e9-bd63-eea3d99408e5",
  "email_verified": false,
  "name": "Admin Admin",
  "preferred_username": "admin",
  "given_name": "Admin",
  "family_name": "Admin",
  "email": "admin@admin.com"
...

So I don't see any fundamental difference.

Reply:
thanks